docs/integrations/ledger/ledger_EVM.md

```diff
@@ -0,0 +1,171 @@
+---
+sidebar_position: 1
+title: Ledger EVM
+---
+
+## Using Astar and Shiden EVM ledger apps with MetaMask
+
+## Intro
+
+“Astar EVM” and “Shiden EVM” apps are now available on Ledger hardware wallet devices. This means that MetaMask users can now sign transactions for EVM accounts on those networks using Ledger nano S (plus) or Ledger X devices.
```

Comment:
I'd remove the quotes and make them bold by surrounding each with *. 

eg. \*\*Astar EVM** and \**Shiden EVM\*\* apps are now..
will look like
 **Astar EVM** and **Shiden EVM** apps are now...